projects/plutus/inc/ltc_afe.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+
+#include <stdbool.h>
+#include <stdint.h>
+#include "gpio.h"
+#include "spi.h"
+#include "status.h"
+
+// number of devices in daisy chain (including master)
+#define LTC_AFE_DEVICES_IN_CHAIN                1
```

Comment:
Config header sounds good to me. We don't have a specific place for them, but I'd lean towards keeping it in the project itself. If you wanted to explicitly add a config folder, adding a new folder to rules.mk is pretty easy.